articles/virtual-desktop/app-attach-faq.md

```diff
@@ -55,6 +55,10 @@ Yes. MSIX app attach doesn't support auto-update for MSIX applications.
 
 All virtual machines (VMs) in a host pool that uses MSIX app attach must have read permissions on the file share where the MSIX images are stored. If it also uses Azure Files, they'll need to be granted both role-based access control (RBAC) and New Technology File System (NTFS) permissions.
 
+## Can I use Azure Active Directory Domain Services (AAD DS) with MSIX app attach?
+
+We do not support AAD DS with MSIX app attach, as the AAD DS computer objects are not synchronised to AAD DS and thus cannot be provided the RBAC permissions required in Azure Files
```

Comment:
```suggestion
MSIX app attach doesn't currently support Azure AD DS. Because Azure AD DS computer objects aren't synchronized to Azure Active Directory (Azure AD), they can't get the required role-based access control (RBAC) permissions for Azure Files.
```

Reply:
The first bit is now correct, but the last sentence has now been changed, and needs to revert to be accurate. This is an adidtional step of "providing" the access the admin needs to do, they don't "get" them

MSIX app attach doesn't currently support Azure AD DS. Because Azure AD DS computer objects aren't synchronized to Azure Active Directory (Azure AD), they can't be provided the required role-based access control (RBAC) permissions for Azure Files.

This is completly accurate and good to go

Reply:
Can we say "The service can't provide them the required RBAC permissions?" "Be provided" is passive voice, and I'd like to avoid that if I can.